src/exact.jl

```diff
@@ -97,5 +97,10 @@ function isdominated(A, b, islb, isfun, k, optimizer_constructor, lb, ub, epsilo
     optimize!(model)
 
     res = objective_value(model)
-    return (islb) ? -res < epsilon : res > -epsilon
+
+    if termination_status(model) == JuMP.MOI.OPTIMAL
+        return (islb) ? -res < epsilon : res > -epsilon
+    else
+        return false
```

Comment:
If the terminaton status is `JuMP.MOI.INFEASIBLE` then `return`ing `false` is the right thing to do but in any other case, we should display an error including the raw status (see https://www.juliaopt.org/JuMP.jl/dev/solutions/#JuMP.raw_status) and the termination status

Reply:
To correct my earlier message:
If it's `INFEASIBLE` then it's redundant so we should return `true`.
If it's `DUAL_INFEASIBLE` (i.e. unbounded`) then we should return `false`.

Reply:
I'm confused.
INFEASIBLE => true
DUAL_INFESAIBLE => false
OPTIMAL => (islb) ? -res < epsilon : res > -epsilon

Correct? What do you mean by displaying an error? Printing or yielding? There's no verbose flag at the moment.

Reply:
Yes
```julia
if status == MOI.INFEASIBLE
    return true
elseif status == MOI.DUAL_INFEASIBLE
    return false
elseif status == MOI.OPTIMAL
    return ...
else
    error("Solver returned status $status: $(raw_status(model).)")
end

Reply:
Bump :)